src/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -2,6 +2,15 @@ import SavedObjectRegistryProvider from 'ui/saved_objects/saved_object_registry'
 import 'ui/pager_control';
 import 'ui/pager';
 import _ from 'lodash';
+import 'ngreact';
+
+import { PlusIcon } from 'ui_framework/components/icon/plus_icon';
+import { TrashIcon } from 'ui_framework/components/icon/trash_icon';
+
+import uiModules from 'ui/modules';
+const app = uiModules.get('app/visualize', ['react']);
+app.value('PlusIcon', PlusIcon);
```

Comment:
I wonder if we can come up with a naming convention that distinguishes react components from all other other values in the injector... maybe `XYZComponent`?